the-badass-todo-list-part-1/list.js

```diff
@@ -52,8 +57,10 @@
   }
 
   // Kick off
+  // Quiz #2
```

Comment:
Yes! There are lots of nasty racings and other bugs drive you crazy caused by the problems similar to this.